src/Compilers/CSharp/Test/Semantic/Semantics/IteratorTests.cs

```diff
@@ -407,5 +407,27 @@ protected override IEnumerable<int> M()
             var comp = CompileAndVerify(source, expectedOutput: "0,1,2,3", options: TestOptions.DebugExe);
             comp.Compilation.VerifyDiagnostics();
         }
-   }
+
+        [Fact]
+        [WorkItem(261047, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=261047&_a=edit")]
+        public void MissingExpression()
```

Comment:
I'm assuming this is no longer reproducible, thus you're adding this test because you have a PR pending. Please let me know if that assumption is incorrect.